Interface/AddOns/nMainbar/modules/short/short.lua

```diff
@@ -81,13 +81,53 @@ for _, object in pairs({
     _G['MainMenuBar'],
     _G['MainMenuExpBar'],
     _G['MainMenuBarMaxLevelBar'],
-
-    _G['ReputationWatchBar'],
-    _G['ReputationWatchStatusBar'],
 }) do
     object:SetWidth(512)
 end
 
+ReputationWatchBar:SetWidth(512)
+ReputationWatchBar.StatusBar:SetWidth(512)
+ReputationWatchBar.OverlayFrame:SetWidth(512)
```

Comment:
I was able to remove the OverlayFrame code but for reason StatusBar isn't inheriting the width and using the _G[] method isn't working either.